webrtc.html

```diff
@@ -2849,6 +2849,16 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       Each track to be sent is associated with exactly one <code><a>RTCRtpSender</a></code>, and
       each track to be received is associated with exactly one <code><a>RTCRtpReceiver</a></code>.</p>
 
+      <p>The basic principle is that the encoding and transmission of each
+      <code>MediaStreamTrack</code> is made such that the characteristics are as far as possible
+      maintained by the track created on the remote side. In other words, the
+      <code>MediaTrackSettings</code> width, height, frameRate for a video track, and volume, sampleSize,
+      sampleRate and channelCount for an audio track should be maintained. There are situations when this
+      does not apply, there may for example be resource constraints (HW Codec, CPU, network throughput
+      etc.) that makes this impossible, or there may be <code><a>RTCRtpSender</a></code> settings applied
```

Comment:
Should we also point out that these characteristics may even vary over time? For instance, this comes up as an issue during remote recording, where not all players cope well with variable-resolution recordings, so it might be useful to mention that there's no guarantee of characteristics being fixed (whatever they are).